pandas-stubs/core/frame.pyi

```diff
@@ -1987,11 +1987,13 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def ne(self, other, axis: Axis = ..., level: Level | None = ...) -> Self: ...
     def pct_change(
         self,
-        periods: int = ...,
+        periods: int = 1,
```

Comment:
You can't do this in a stubs.  Please leave it as `periods: int = ...` because the `periods` argument is optional.

Reply:
Sorry, could you please clarify? The argument is still optional when written like this.

I see a few other places where the default is written when it's a simple literal, like https://github.com/pandas-dev/pandas-stubs/blob/main/pandas-stubs/core/frame.pyi#L317

Reply:
Understood, thank you